consensus/consensus.go

```diff
@@ -77,7 +77,6 @@ type Consensus struct {
 	// blockNum: the next blockNumber that FBFT is going to agree on,
 	// should be equal to the blockNumber of next block
 	blockNum uint64
-	viewID   uint64
```

Comment:
This view ID is not redundant. This is the viewID for the last successful consenus. The current.viewID is the moving viewID during view change. They can not be merged together, at least based on the current logic. Things will break if we don't keep track of them separately in current code.

Reply:
wow! that's a precious hidden knowledge. I can see why it is very confusing.

Reply:
yup. The whole view change code logic is not clean. Very error-prone.